impl/src/checks.rs

```diff
@@ -166,4 +237,19 @@ mod tests {
     // Should not panic with valid metadata.
     check_resolve_matches_packages(&metadata).unwrap();
   }
+
+  #[test]
+  fn test_verifies_vendored_state() {
```

Comment:
This test doesn't follow the naming scheme of tests in this module, which is

test_$FUNCTION_UNDER_TEST_$HAS_BEHAVIOR.